can-view-target.js

```diff
@@ -111,7 +111,12 @@ var cloneNode = clonesWork ?
 			for (var i = 0; i < attributes.length; i++) {
 				var attribute = attributes[i];
 				if (attribute && attribute.specified) {
-					domMutate.setAttribute.call(copy, attribute.nodeName || attribute.name, attribute.nodeValue || attribute.value);
+					// If the attribute has a namespace set the namespace 
+					// otherwise t will be set to null
+					if (attribute.namespaceURI) {
+						copy.setAttributeNS(attribute.namespaceURI, attribute.nodeName || attribute.name, attribute.nodeValue || attribute.value);
+					}
+					node.setAttribute.call(copy, attribute.nodeName || attribute.name, attribute.nodeValue || attribute.value);
```

Comment:
Should this be in an `else`? If you set the attribute with `setAttributeNS`, does it also need to be set with `setAttribute`?